releasetool/github.py

```diff
@@ -13,16 +13,23 @@
 # limitations under the License.
 
 import base64
+import json
 import os
 import re
-from typing import List, Sequence, Union
+import time
 
+from typing import cast, Any, List, Sequence, Union
+
+import jwt
 import requests
+from cryptography.hazmat.backends import default_backend
 
 
 _GITHUB_ROOT: str = "https://api.github.com"
 _GITHUB_UI_ROOT: str = "https://github.com"
-_MAGIC_GITHUB_PROXY_ROOT: str = "https://magic-github-proxy.endpoints.devrel-prod.cloud.goog"
+_MAGIC_GITHUB_PROXY_ROOT: str = (
```

Comment:
Do we still need the magic proxy?

Reply:
I was thinking about tearing out magic proxy code, but it's pretty interleaved, and I thought perhaps it would be good to do in a future PR, once we've proven this approach.